recipes/distributed/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "1.10.1" %}
+
+package:
+  name: distributed
+  version: "{{ version }}"
+
+source:
+  fn: distributed-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/distributed/distributed-{{ version }}.tar.gz
+  md5: 98c6ec894386a93ffd16718d04b79671
+
+build:
+  number: 0
+  entry_points:
+    - dcenter=distributed.cli.dcenter:go
+    - dscheduler=distributed.cli.dscheduler:go
+    - dcluster=distributed.cli.dcluster:start
+    - dworker=distributed.cli.dworker:go
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - tornado >=4.2
+    - toolz
+    - msgpack-python
+    - cloudpickle >=0.2.1
+    - dask >=0.9.0
+    - click
+    - boto3
+    - s3fs
+    - locket
+    - six
+    - tblib
+    - psutil
+
+  run:
+    - python
+    - tornado >=4.2
+    - toolz
+    - msgpack-python
+    - cloudpickle >=0.2.1
+    - dask >=0.9.0
+    - click
+    - boto3
+    - s3fs
+    - locket
+    - six
+    - tblib
+    - psutil
+    - bokeh  # [not py26]
+
+test:
+  imports:
+    - distributed
+    - distributed.bokeh
+    - distributed.bokeh.status
+    - distributed.bokeh.tasks
+    - distributed.cli
+    - distributed.diagnostics
+    - distributed.http
+  commands:
+    - dcenter --help
+    - dscheduler --help
+    - dcluster --help
+    - dworker --help
+
+about:
+  home: http://distributed.readthedocs.io/en/latest/
+  license: BSD 3-Clause
+  summary: 'Distributed computing'
+
+extra:
+  recipe-maintainers:
+    - mrocklin
```

Comment:
Sorry, should have done this before. Could you please confirm that you are happy to be a maintainer, @mrocklin?